File: Tshirt.java
package com.sonata.day2;

public class Tshirt {

		String color;
		String material;
		String design;
		public Tshirt(String color, String  material, String design)
		{
			this.color = color;
			this. material =  material;
			this.design = design;
		}
		public void display()
		{
			System.out.println(color+" "+material+" "+design);
		}
		public static void main(String[] args)
		{
			Tshirt small = new Tshirt("violet", " cotton", "half sleev");
			Tshirt large = new Tshirt("black", " silk", "round neck");
			Tshirt ex_large = new Tshirt("white", " semi cotton", "full sleev");
			small .display();
			large .display();
			ex_large .display();
		}

	}

File: JavaScript/TShirt.js
var TShirt = /** @class */ (function () {
    function TShirt(Color, Material, Design) {
        this.Color = Color;
        this.Material = Material;
        this.Design = Design;
    }
    TShirt.prototype.displayName = function () {
        console.log(" The Color of the Shirt is  " + this.Color + ", Material is " + this.Material + " , Design is  " + this.Design);
    };
    return TShirt;
}());
var small = new TShirt("Violet", "Silk", "Full Sleev");
small.displayName();
var Large = new TShirt("White", "Cotton", "Full Neck");
Large.displayName();
var ExLarge = new TShirt("Red", "Semi Cotton", "Neck");
ExLarge.displayName();
